test/integration/volumescheduling/volume_binding_test.go

```diff
@@ -946,10 +945,11 @@ func initPVController(context *testContext, provisionDelaySeconds int) (*persist
 	}
 	plugins := []volume.VolumePlugin{plugin}
 
-	controllerOptions := persistentvolumeoptions.NewPersistentVolumeControllerOptions()
 	params := persistentvolume.ControllerParameters{
-		KubeClient:                clientset,
-		SyncPeriod:                controllerOptions.PVClaimBinderSyncPeriod,
+		KubeClient: clientset,
+		// Use a frequent resync period to retry API update conflicts due to
+		// https://github.com/kubernetes/kubernetes/issues/85320
+		SyncPeriod:                5 * time.Second,
```

Comment:
this takes the attempts from 45 seconds to 15, assuming 3 retries are needed?

Reply:
Is it still worth waiting for the PVs to be marked available before creating the PVCs in TestVolumeBinding?

Reply:
10 seconds, assuming 1st conflict occurs at time 0. test timeout is 30 seconds. I could check for PVs to be available too, which would take the worse-case wait down to 5 seconds

Reply:
added one more commit to wait for PVs to be available